recipe/meta.yaml

```diff
@@ -25,6 +27,11 @@ requirements:
   run:
     - python
 
+test:
+  commands:
+    - dir %PREFIX%\pythoncom%CONDA_PY%.dll
```

Comment:
(previously I meant "bat" files obviously)

OK thanks, I will change the recipe.

Although it did not work so far: https://ci.appveyor.com/project/conda-forge/pywin32-feedstock/build/1.0.7/job/0gn0gky0nfgp22f3

Do you know what might be the problem at a glance? Otherwise I will try this when I have some more time.